codeflash/discovery/discover_unit_tests.py

```diff
@@ -37,13 +39,100 @@ class TestFunction:
 FUNCTION_NAME_REGEX = re.compile(r"([^.]+)\.([a-zA-Z0-9_]+)$")
 
 
+class TestsCache:
+    def __init__(self) -> None:
+        self.connection = sqlite3.connect(codeflash_cache_dir / "tests_cache.db")
```

Comment:
can you make `tests_cache.db` into a general `codeflash_cache.db`?
We want to cache other things in the future too, we can avoid breaking changes in the future with this